setup/windows.md

```diff
@@ -1,7 +1,49 @@
-
 # Windows Setup
+The workshop requires the following:
+1. Use of a `shell`, the `Command Prompt` will do,
+2. Java JDK (Java Development Kit) installed,
+3. Scala installed, and
+4. A text editor.
+
+## Windows Shell
+1. Press the `Windows` key and the `E` key together to open a Windows Explorer window with `Computer` selected.
+2. Right click on `Computer` and select `Properties` from the context menu. This opens the System Control Panel window. Find the `System type:` field and note whether your computer is running 64-bit or 32-bit Windows.
+3. Close the System Control Panel window.
+4. Return to the Windows Explorer window. It should show `Computer` in the Address Bar toward the top of the window:
+
+![](/img/address_bar.png)
```

Comment:
Looks like all the images referenced in this file need to have `setup` prepended to their paths. See if `setup/img/address_bar.png` works here?